apps/components/src/app/toolbar/toolbar.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { NgpToolbar } from 'ng-primitives/toolbar';
+
+@Component({
+  selector: '[app-toolbar]',
+  hostDirectives: [{ directive: NgpToolbar, inputs: ['ngpToolbarOrientation'] }],
```

Comment:
We can rename the input here for brevity on the component.

```suggestion
  hostDirectives: [{ directive: NgpToolbar, inputs: ['ngpToolbarOrientation:orientation'] }],
```